test/e2e/serial/config/config.go

```diff
@@ -165,15 +165,9 @@ func CheckNodesTopology(ctx context.Context) error {
 		return fmt.Errorf("Following nodes have incoherent info in KubeletConfig/NRT data:\n%s\n", string(prettyMap))
 	}
 
-	//TODO - Can look just for single-numa-node policy whatever the scope
-	snnPodLevelNRTs := e2enrt.FilterByTopologyManagerPolicyAndScope(nrtList.Items, intnrt.SingleNUMANode, intnrt.Pod)
-	numSnnPodLevelNRTs := len(snnPodLevelNRTs)
-
-	snnContainerLevelNRTs := e2enrt.FilterByTopologyManagerPolicyAndScope(nrtList.Items, intnrt.SingleNUMANode, intnrt.Container)
-	numSnnContainerLevelNRTs := len(snnContainerLevelNRTs)
-
-	if numSnnPodLevelNRTs < minNumberOfNodesWithSameTopology && numSnnContainerLevelNRTs < minNumberOfNodesWithSameTopology {
-		return fmt.Errorf("Not enough nodes with either %q topology (found:%d) nor %q topology (found: %d). Need at least %d", nrtv1alpha2.SingleNUMANodePodLevel, numSnnPodLevelNRTs, nrtv1alpha2.SingleNUMANodeContainerLevel, numSnnContainerLevelNRTs, minNumberOfNodesWithSameTopology)
+	singleNUMANodeNRTs := e2enrt.FilterByTopologyManagerPolicy(nrtList.Items, intnrt.SingleNUMANode)
+	if len(singleNUMANodeNRTs) < minNumberOfNodesWithSameTopology {
+		return fmt.Errorf("Not enough nodes with %q topology (found:%d). Need at least %d", intnrt.SingleNUMANode, len(singleNUMANodeNRTs), minNumberOfNodesWithSameTopology)
```

Comment:
looks good: please squash in the parent commit - unless there's a clear reason (which I'm missing atm) to keep it separate. If so please describe in the commit message.

Reply:
I just kept it in a different commit because it is just a minor improvement in a test using the newly created filter functions

Reply:
In this case I'm leaning towards squashing - we're mixing mechanical translations with changes anyway, and it's OK (if done sparingly)